plugins/de/opendiabetes/vault/plugin/exporter/plotterexporter/help.md

```diff
@@ -0,0 +1,41 @@
+# PlotterExporter
+Version: 1.0.0  
+Classification: Exporter
+
+Overview
+-----
+The PlotterExporter is an exporter plugin which exports data from the database. It is responsible for the provision of data for the plotter. After the data was exported to a valid CSV file using the CSVExporter, the file is used to create plot from the data. After the process is finished the exported CSV file will be deleted.
+
+It is possible to export the data into a PNG or PDF file. 
+In order for the exporter to work the environment needs to have [Python 2.3](https://www.python.org/download/releases/2.3/) installed.
+Also, the following libraries are required in the given version:
+
+- [matplotlib (2.0.0)](https://matplotlib.org/2.0.0/index.html)
+- [configparser](https://docs.python.org/2/library/configparser.html)
+
+
+If you want to export the plot in a PDF file, the environment needs [pdflatex](https://www.latex-project.org/get/#tex-distributions) to be installed additionally.
```

Comment:
Klingt komisch, würde _... additionally pdflatex needs to be installed in the environment_ vorschlagen